.github/dependabot.yml

```diff
@@ -40,6 +40,22 @@ updates:
       - dependency-name: github.com/elastic/mito
     open-pull-requests-limit: 2
 
+  # Azure SDKs
+  - package-ecosystem: "gomod"
+    directory: "/"
+    ignore:
+      - dependency-name: "github.com/elastic/beats/v7"
```

Comment:
I guess you got this from apm-server? I don't think this is relevant here

Reply:
Yeah, my apologies for the noise 🤦

Reply:
all good :)